iquip/apps/scheduler.py

```diff
@@ -59,6 +81,22 @@ def __init__(self, parent: Optional[QWidget] = None):
         layout.addWidget(self.queueView)
 
 
+class ExperimentListView(QListView):
+    """Customized QListView class to detect right-click input.
+    
+    Signals:
+        rightButtonPressed(QMouseEvent): The information of the click input is sent.
+    """
+    rightButtonPressed = pyqtSignal(QMouseEvent)
+
+    def mousePressEvent(self, event):
+        """Overridden."""
+        if event.type() == QEvent.MouseButtonPress:
+            if event.button() == Qt.RightButton:
```

Comment:
Why don't you use `and` instead of nesting 2 `if` statements? Is it for future use?